sdk/batch/Microsoft.Azure.Batch/src/GeneratedProtocol/Models/PoolEndpointConfiguration.cs

```diff
@@ -52,7 +52,8 @@ public PoolEndpointConfiguration(IList<InboundNATPool> inboundNATPools)
         /// <remarks>
         /// The maximum number of inbound NAT Pools per Batch Pool is 5. If the
         /// maximum number of inbound NAT Pools is exceeded the request fails
-        /// with HTTP status code 400.
+        /// with HTTP status code 400. This cannot be specified if the
+        /// IPAddressProvisioningType is NoPublicIPAddresses.
```

Comment:
Is IPAddressProvisioningType an enum we can link to?

Reply:
This model is purely autogenerated from swagger. Generally speaking it is best to update the actual Swagger defined RestAPI documentation, because any change here will just get replaced each generation.